docs/framework-statistics.md

```diff
@@ -0,0 +1,30 @@
+<!--
+title: Serverless framework statistics
+menuText: Framework statistics
+layout: Doc
+-->
+
+# Framework statistics
+
+Serverless will automatically collect anonymous framework statistics. This is done so that we better understand the usage and needs
+of our users to improve Serverless in future releases. However you can always [disable it](#how-to-disable-it).
+
+## What we collect
+
+Our main goal is anonymity. All the data is anonymized and won't reveal who you are or what the project you're working on is / looks like.
+
+Please take a look at the [`logStat()` method](../lib/classes/Utils.js) in the `Utils` class to see what (and how) we collect statistics.
```

Comment:
absolute URL is needed here. See https://github.com/serverless/serverless/pull/2369